### PR TITLE
600 email

### PR DESCRIPTION
**Pull request by mtlaney**
## Summary
Addresses Issue #600

Please note if fully resolves the issue per the acceptance criteria: Yes

## Impacted Areas of the Site
- Front End Email Validation

## This pull request changes...
- [x] Throws and error when entering an email in aaaaa.aa fashion
- [x] Changes email regex pattern to match user@email.com type patterns

## This pull request is ready to merge when...
- [x] Feature branch is starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [x] Tests have been updated 
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [x] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/src/app/application-forms/validators/error-message.component.spec.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ describe('ErrorMessageComponent', () => {
       min: ['', Validators.min(1)]
     });
     form.get('emailAddress').setValue('test');
-    expect(component.parseErrors(form.get('emailAddress').errors)).toEqual('Test requires a valid email address. ');
+    expect(component.parseErrors(form.get('emailAddress').errors)).toEqual('Test requires a valid email address. I.E. you@email.com ');
```
Comment (davemcorwin) [inline frontend/src/app/application-forms/validators/error-message.component.spec.ts]: @jdamis4XD @DawnDesigner @lisaredux Is this change to the invalid email error message ok?

Comment (lisaredux) [reply frontend/src/app/application-forms/validators/error-message.component.spec.ts]: Thanks Dave. I'm not sure I see the update. 

I don't have a problem with the actual message — just that it comes too early before someone has clicked into that text input field. Ideally should show when someone clicks out of that field.

Screenshot 2019-05-07 09 52 00

I defer to Amber about the content of the message itself.

Comment (davemcorwin) [reply frontend/src/app/application-forms/validators/error-message.component.spec.ts]: @lisaredux the email confirmation validation is a different issue (#489 ) Here, Mike made a change to the **message** the user sees when an invalid email address is entered and I just want to make sure that it is ok with y'all.

## Before
`Email requires a valid email address.`

## After
`Email requires a valid email address. I.E. you@email.com`

Comment (lisaredux) [reply frontend/src/app/application-forms/validators/error-message.component.spec.ts]: Thanks @davemcorwin  — I see now. Thanks for clarifying. 
I recommend: "Requires a valid email address— i.e. you@email.com"
Though I'm going to defer to @ASprinkle on the wording and standards.

Comment (ASprinkle) [reply frontend/src/app/application-forms/validators/error-message.component.spec.ts]: Yes, the timing piece is a different story.  Agreed, "Requires a valid email address" when an invalid email address is entered.  There should be a separate error note for when the email verification entered does not match... Correct?

Comment (davemcorwin) [reply frontend/src/app/application-forms/validators/error-message.component.spec.ts]: @ASprinkle Yes, and there is. No changes are made here to the email confirmation validation error.

Comment (mtlaney) [reply frontend/src/app/application-forms/validators/error-message.component.spec.ts]: @lisaredux sounds like we want it to read: Requires a valid email address— i.e. you@email.com
as opposed to: Email requires a valid email address. I.E. you@email.com.

I'll get that change implemented before the merge

**Pull request merged**